File: Automator/src/main/java/com/winkar/SingleAppTester.java
package com.winkar;

import org.apache.log4j.Logger;

/**
 * Created by WinKaR on 16/3/18.
 */
public class SingleAppTester {
    static Logger log = Logger.getLogger(Automator.class.getName());
    private final String apkPath;

    SingleAppTester(String apkPath) {
        this.apkPath = apkPath;
    }

    public void startTest() {

    }
}

File: Automator/src/main/scala/com/winkar/SingleAppTester.scala
package com.winkar

import org.apache.log4j.Logger


class SingleAppTester private[winkar](val apkPath: String) {
  val log: Logger = Logger.getLogger(Automator.getClass.getName)
  def startTest: Unit = {
    new AppTraversal(apkPath).start
  }
}
